packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -567,6 +567,13 @@ export interface UserPoolProps {
    * @default AccountRecovery.PHONE_WITHOUT_MFA_AND_EMAIL
    */
   readonly accountRecovery?: AccountRecovery;
+
+  /**
+   * Policy to apply when the user pool is removed from the stack
+   *
+   * @default - The user pool will be orphaned.
```

Comment:
assumed these `@default` comments themselves were setting defaults since I don't see the default being set anywhere else in `lib/`, so this one confuses me. Not sure how this gets `Cognito User Pools (default: RETAIN)` from the PR description, should it be something like this?

```suggestion
   * @default RemovalPolicy.RETAIN
```